src/views/editProfile.js

```diff
@@ -1,8 +1,23 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getProfile } from '../redux/actioncreators/userProfile';
 import EditProfileForm from '../components/Profile/EditProfileForm';
+import Loader from '../components/Loader/Loader';
 
 export class EditProfile extends Component {
+  componentDidMount() {
+    const { getProfile } = this.props
+    getProfile()
+  }
   render() {
+    const { isFetching } = this.props;
+    if (isFetching === true) {
```

Comment:
Just check on ```isFetching```, no need for checking with an equal operator.

Reply:
I was not doing the profile feature, I think the comments are for Kevin's feature

Reply:
Yes, i think, @Arusey work should be merged, then you get the latest changes and rebase them.